### ISSUE TITLE
Update Chart.JS to 2.4 / use a different chart library

### ISSUE THREAD

**Issue by aikar** [aikar/timings#30]
Once the New UI is done, would love if someone could PR this.

**Comment by Slind14** [comment]
https://d3js.org/ Would be my pick, but it is low level.

**Comment by aikar** [comment]
yeah im looking at charts specifically, data binding to dom is handled by React.

**Comment by aikar** [comment]
based on latest research at work, the latest version of Chart.js rocks and solved everyone's problems with 1.

We chose to go with it at work out of all the options, so should stick to it just use latest.
